deploy-olm-operator.yml

```diff
@@ -48,6 +49,10 @@
         name: inject_scorecard_metadata
       when: run_scorecard|bool
 
+    - name: "Set the deployment start time"
+      set_fact:
+        start_time: "{{ ansible_date_time.iso8601 }}"
+
```

Comment:
@dirgim IIUC we will very likely use different role for deploying OLM operator for operator bundles.
Is my assumption correct?

Reply:
@lslebodn that's correct, this deployment role will be used exclusively for the legacy operator metadata format.
That said, I see no harm of collecting the pulled images as part of this role to provide the legacy pipeline with the same functionality as the new one will have.

Reply:
And I see no benefit of collecting them. But I am looking forward for reasonable justification of this change.

Reply:
@lslebodn By making this change, we keep all of the code for the image source test downstream the same. I think there's value in that

Of less importance, if we don't implement this here, the ticket associated with this will be blocked until the new playbook is created.

Reply:
> @lslebodn By making this change, we keep all of the code for the image source test downstream the same. I think there's value in that

I do not follow here.

It will be just a dead code and increase test-time without any value.

Reply:
>  if we don't implement this here, the ticket associated with this will be blocked until the new playbook is created.

Ticket is blocked anyway because the scorecard test for operator bundles does not exist yet.
Creating role is useful and should be merged but we will not be able to resolve issue anyway.